src/api/spec/jobs/measurements_job_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe MeasurementsJob do
+  describe 'token workflows' do
+    it 'gathers some intelligence about our token workflow runs' do
+      allow(RabbitmqBus).to receive(:send_to_bus)
```

Comment:
```suggestion
    it 'gathers some intelligence about our token workflow runs' do
      stub_const('CONFIG', CONFIG.merge('amqp_options' => { dummy: 1 }))
      allow(RabbitmqBus).to receive(:send_to_bus)
```

We should give the configirutation key `amqp_options` some value...

Reply:
Yeah, I figured it out